app/styles/libs/_helpers.scss

```
@@ -127,6 +127,30 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
   width: fit-content;
 }
 
+.expand-bar {
+  width: 10px;
+  height: 36px;
+
+  &:focus {
+    width: 100%;
+  }
+}
+
+.show-div {
+  display: none;
+}
+
+.full-width {
+  width: 100%;
+}
+
+.time-zone-picker-text {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  max-width: 88px;
```

fix the sorting lint of scss. 
```suggestion
  max-width: 88px;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
```